controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -37,24 +37,16 @@ static const float COMPARISON_TOLERANCE = 0.005f;
 // Maximum allowable delta between calculated and actual volumetric flow
 static const float COMPARISON_TOLERANCE_FLOW = 5.0e-5f;
 
-// ADC of Nano is 10 bit, 5V VREF_P [ADC Counts / V]
-static const float COUNTS_PER_VOLT = 1024.0f / 5.0f;
-
 //@TODO: Finish writing more specific unit tests for this module
 
 /*
  * @brief This method models the pressure to voltage transfer function of the
- * MPXV5004 series sensors.
+ * MPXV5004 series sensors.  The raw voltage coming out of the sensor would be
+ * 5 * (...), but our PCB scales it down to 3.3f * (...) so that the pressure
+ * range (0-4kPa) is in the voltage range 0-3.3V.
  */
 static Voltage MPXV5004_PressureToVoltage(Pressure pressure) {
-  return volts(5 * (0.2f * pressure.kPa() + 0.2f));
-}
-
-// Simple helper function that takes in a voltage and returns the
-// equivalent ADC counts that represent it
-static void test_setAnalogPinToVolts(AnalogPin pin, Voltage v) {
-  Hal.test_setAnalogPin(pin,
-                        static_cast<int>(roundf(v.volts() * COUNTS_PER_VOLT)));
+  return volts(3.3f * (0.2f * pressure.kPa() + 0.2f));
```

Comment:
The 0.2 offset captures the voltage offset at 0 differential pressure that I mentioned above.